### DISCUSSION TITLE
My first blog! Thanks!!

### DISCUSSION THREAD

**Discussion by wzrdk3lly** [jpanther/congo#49] [Show and tell]
Thank you to @jpanther for answering all of my questions when learning how to use congo+hugo. I created my first blog to document my tech journey. check it out @ https://mkduke424.github.io/myblog/

**Comment by wzrdk3lly** [comment]
feel free to provide any feedback on how I can improve the set up. Thanks again!

**Reply by AntoineSoetewey** [reply]
Looks great!

just one remark:

![CBBD668E-2AEE-4062-96A0-B53DE5027748](https://user-images.githubusercontent.com/17910063/145269193-be22e1cd-f562-4b60-bb1e-108caa7c5e0d.png)

Due to the long string for your bio, it doesn’t fit on mobile

**Reply by wzrdk3lly** [reply]
Thanks @AntoineSoetewey! I just updated the string in the author bio.

**Comment by jpanther** [comment]
Thanks @mkduke424! If you like, feel free to add your site to the [users page](https://jpanther.github.io/congo/users/).
